src/app/pages/SearchResultsPage/hooks.ts

```diff
@@ -234,6 +235,21 @@ export function useNamedAccountConditionally(
   }
 }
 
+export function useNamedValidatorConditionally(nameFragment: string | undefined) {
+  const queries = RouteUtils.getEnabledNetworksForLayer(Layer.consensus).map(network =>
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    useSearchForValidatorByName(network, nameFragment),
```

Comment:
Don't particularly like this, it should be something like `useSearchForValidatorByNames`, which would receive an array of networks.

Reply:
Name updated, but I will keep that pattern where we create a hook just to brake a rule of hooks. This is how entire search was built. Will create an issue to refactor all at once.

Reply:
https://github.com/oasisprotocol/explorer/issues/1635